dockerfiles/ci/xfail_tests/7.4.list

```diff
@@ -392,3 +392,18 @@ ext/standard/tests/streams/bug78902.phpt
 ext/phar/tests/phar_buildfromiterator10.phpt
 ext/phar/tests/phar_buildfromiterator8.phpt
 ext/standard/tests/file/lstat_stat_variation10.phpt
+sapi/fpm/tests/log-bm-in-shutdown-fn.phpt
+sapi/fpm/tests/log-bm-limit-1024-msg-80.phpt
+sapi/fpm/tests/log-bm-limit-2048-msg-4000.phpt
+sapi/fpm/tests/log-bwd-limit-1050-msg-2048.phpt
+sapi/fpm/tests/log-bwd-limit-1050-msg-2900.phpt
+sapi/fpm/tests/log-bwd-limit-8000-msg-4096.phpt
+sapi/fpm/tests/log-bwd-msg-with-nl.phpt
+sapi/fpm/tests/log-bwp-limit-1024-msg-120.phpt
+sapi/fpm/tests/log-bwp-limit-1500-msg-3300.phpt
+sapi/fpm/tests/log-bwp-msg-flush-split-fallback.phpt
+sapi/fpm/tests/log-bwp-msg-flush-split-real.phpt
+sapi/fpm/tests/log-dwd-limit-1050-msg-2048.phpt
+sapi/fpm/tests/log-dwd-limit-1050-msg-2900.phpt
+sapi/fpm/tests/log-dwd-limit-8000-msg-4096.phpt
+sapi/fpm/tests/log-dwp-limit-1000-msg-2000.phpt
```

Comment:
Since the startup logs now happen in RINIT, there is no way to pass `DD_TRACE_STARTUP_LOGS=0` into these tests since FPM will ignore system env vars by default.